sklearn/metrics/_classification.py

```diff
@@ -218,6 +230,16 @@ def accuracy_score(y_true, y_pred, *, normalize=True, sample_weight=None):
     0.5
     """
 
+    # Check y_true and y_pred is empty
+    len_y_true = _num_samples(y_true)
+    len_y_pred = _num_samples(y_pred)
+
+    if len_y_true == 0 and len_y_pred == 0:
+        score = _check_zero_division(zero_division)
+        if zero_division == "warn":
+            _warn_prf(None, "Predcited", "Accuracy is", 0)
```

Comment:
```suggestion
            _warn_prf(None, "Predcited", "`accuracy_score` is", 0)
```
I think we should be precise and tell our user where exactly the warning stems from.

Reply:
Done